newm-server/src/main/kotlin/io/newm/server/features/release/repo/OutletReleaseRepositoryImpl.kt

```diff
@@ -72,12 +72,12 @@ internal class OutletReleaseRepositoryImpl(
                     }
                     contentType(ContentType.Application.Json)
                     accept(ContentType.Application.Json)
-                    setBody {
+                    setBody(
                         SpotifyRequest(
-                            arrayOf(trackUri),
-                            0
+                            uris = listOf("spotify:track:$trackUri"),
```

Comment:
Why the "spotify:track:" prefix? Was that another fix?

Reply:
I believe you can add things to playlist by other things such as an ISRC code maybe. This lets spotify know that you're providing a trackUri.